blocks/form/mappings.js

```diff
@@ -1,7 +1,7 @@
 import { loadCSS } from '../../scripts/aem.js';
 
-let customComponents = [];
-const OOTBComponentDecorators = ['file-input', 'wizard', 'modal', 'tnc', 'toggleable-link', 'rating', 'datetime', 'list', 'location', 'accordion', 'password'];
```

Comment:
any reason we are removing datetime, list and location components?

Reply:
They were never custom components, their names were present erroneously as verified.